packages/web-frontend/src/sagas.js

```diff
@@ -704,22 +699,22 @@ function* fetchViewData(parameters, errorHandler) {
   const { startDate, endDate } =
     parameters.startDate || parameters.endDate
       ? parameters
-      : getDefaultDates(state.global.viewConfigs[infoViewKey]);
+      : getDefaultDates(state.global.viewConfigs[infoViewKey] || {});
+
   // Build the request url
   const {
     organisationUnitCode,
     dashboardGroupId,
     viewId,
-    drillDownLevel,
```

Comment:
Not a functional change, drillDownLevel is in extraUrlParameters now (but no strong preference on this)